README.md

```diff
@@ -29,42 +141,59 @@ Your app is ready to be deployed!
+👤 **Diego Luna**
 
-You don't have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn't feel obligated to use this feature. However we understand that this tool wouldn't be useful if you couldn't customize it when you are ready for it.
+- GitHub: [@SlowKingV](https://github.com/SlowKingV)
+- Twitter: [@SlowKingVI](https://twitter.com/SlowKingVI)
+- LinkedIn: [Diego Luna Granados](https://www.linkedin.com/in/diego-luna-granados/)
 
-## Learn More
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-You can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
+<!-- FUTURE FEATURES -->
 
-To learn React, check out the [React documentation](https://reactjs.org/).
+## 🔭 Future Features <a name="future-features"></a>
 
-### Code Splitting
+- [ ] **Do the calculator**
+- [ ] **Create the Quote page**
+- [ ] **Make it choose a random quote on every reload**
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/code-splitting](https://facebook.github.io/create-react-app/docs/code-splitting)
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-### Analyzing the Bundle Size
+<!-- CONTRIBUTING -->
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size](https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size)
+## 🤝 Contributing <a name="contributing"></a>
 
-### Making a Progressive Web App
+Contributions, issues, and feature requests are welcome!
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app](https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app)
+Feel free to check the [issues page](../../issues/).
 
-### Advanced Configuration
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/advanced-configuration](https://facebook.github.io/create-react-app/docs/advanced-configuration)
+<!-- SUPPORT -->
 
-### Deployment
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project, please consider leaving a star as this encourages me to keep working and improving it!
+Also if you have any feedback/suggestion don't hesitate to leave a comment in the [issues page](../../issues/)!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+- Thanks to Microverse and all my teammates!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/deployment](https://facebook.github.io/create-react-app/docs/deployment)
+## 📝 License <a name="license"></a>
 
-### `npm run build` fails to minify
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] Great job working on your project documentation! 🚀 However, your LICENSE link is not working because you don't have one at the moment. Please follow [GitHub's official docs](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) in order to add a license to your repo! Microverse suggests that you use the MIT one, but feel free to choose. 🤓 🔧